how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -1,64 +1,82 @@
-Kata Containers [versions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml) file.
+> **Note:** For information about the supported versions of these components,
+> see the  Kata Containers
+> [versions.yaml](https://github.com/kata-containers/runtime/blob/master/versions.yaml)
+> file.
 
-## Install containerd(with CRI plugin enabled)
+## Install containerd with CRI plugin enabled
 
-Follow the instructions from [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md)
+- Follow the instructions from the
+  [CRI installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md).
 
-<!---
-```bash
-# Check if containerd is installed
-$ command -v containerd
-```
---->
-
-## Install Kata Containers 
+- Check if `containerd` is now available
+  ```bash
+  $ command -v containerd
+  ```
 
-Follow the instructions to [install Kata](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+## Install Kata Containers
 
-<!---
-```bash
-# Check if kata-runtime is installed
-$ command -v kata-runtime
-# Check kata is well configured
-$ kata-runtime kata-env
-```
---->
+Follow the instructions to
+[install Kata Containers](https://github.com/kata-containers/documentation/blob/master/install/README.md).
 
 ## Install Kubernetes
-Install Kubernetes in your host.  See kubeadm [installation](https://kubernetes.io/docs/tasks/tools/install-kubeadm/)
-<!---
-```bash
-# Check if kubadm is installed
-$ command -v kubeadm
-```
---->
 
-### Configure containerd to use Kata Containers
+- Follow the instructions for
+  [kubeadm installation](https://kubernetes.io/docs/setup/independent/install-kubeadm/).
+
+- Check `kubeadm` is now available
+
+  ```bash
+  $ command -v kubeadm
+  ```
+
+## Configure containerd to use Kata Containers
 
-The CRI containerd plugin support configuration for two runtime types.
+The CRI `containerd` plugin supports configuration for two runtime types.
 
-- Default runtime: A runtime that is used by default to run workloads.
-- Untrusted workload runtime: A runtime that will be used run untrusted workloads.
+- **Default runtime:**
 
-#### Define the Kata runtime as `untrusted_workload_runtime` 
+  A runtime that is used by default to run workloads.
 
-Configure the Kata runtime for untrusted workload with the [config option](https://github.com/containerd/cri/blob/v1.0.0-rc.0/docs/config.md)
-`plugins.cri.containerd.untrusted_workload_runtime`.
+- **Untrusted workload runtime:**
+
+  A runtime that will be used to run untrusted workloads. This is appropriate
+  for workloads that require a higher degree of security isolation.
+
+#### Define the Kata runtime as the untrusted workload runtime
+
+Configure `containerd` to use the Kata runtime to run untrusted workloads by
+setting the `plugins.cri.containerd.untrusted_workload_runtime`
+[config option](https://github.com/containerd/cri/blob/v1.0.0-rc.0/docs/config.md):
```

Comment:
The link here points to a specific, and old, version of containerd. Is that deliberate? Feels wrong. Probably should either point at the [master branch](https://github.com/containerd/cri/blob/master/docs/config.md) - or the version that matches the one in your versions.yaml ?  I vote for master.